Jenkinsfile

```diff
@@ -284,6 +284,12 @@ pipeline {
     CHANGE_BRANCH = "${env.CHANGE_BRANCH != null ? env.CHANGE_BRANCH : BRANCH_NAME}"
     CHANGE_TARGET = "${env.CHANGE_TARGET != null ? env.CHANGE_TARGET : BRANCH_NAME}"
     CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo'
+    // Required by the jx preview command: https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/preview/preview.go#L1007
+    ORG = "nuxeo"
+    // Required by the jx preview command: https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/preview/preview.go#L1015
+    APP_NAME = "nuxeo"
```

Comment:
Since we're not using `jx preview` anymore, can we remove `ORG` and `APP_NAME`?

Reply:
In the end you got those out of the way :p